### DISCUSSION TITLE
与时同行与开往合作

### DISCUSSION THREAD

**Discussion by linlinzzo** [travellings-link/travellings#1061] [投票]
事项：是否同意将与时同行作为开往的官方站点，并为开往增加新的功能，与时同行改名为开往。

**Comment by imjason** [comment]
其他意见在评论提出

**Comment by vicevolf** [comment]
作为开往的发起者，我个人表示支持现在的开往维护团队，而非某个方案。

但目前看来，团队内部并没达成一致。若最终仍无一致的方案，我优先支持保守的方案，风险最低也对大家影响最小。

**Reply by xiaoRran** [reply]
与时同行内测站点：https://times.sady0.com

**Reply by xiaoRran** [reply]
具体新平台，非当前模式

**Comment by linlinzzo** [comment]
我不同意开往与【与时同行】合作，并与正方进行了交谈。正方认为将开往的维护工作从 Github 转移到与时同行可以节省精力，有更多时间来维护论坛（见图）。
![1](https://user-images.githubusercontent.com/79007796/176809624-e0ff48f6-d5a8-4363-8c60-b7753ac46b95.PNG)
然而我认为与其与尚未成熟的与时同行合作，不妨与成熟的十年之约合作，并且十年之约有相对成熟的站长论坛，免得论坛还要重新弄起。

就我个人而言，我维护这几个月的开往并没有考虑到手动添加浪费精力的问题。反而将申请、审核功能移到与时同行，个人感觉不太妙。

**Comment by linlinzzo** [comment]
![O}`9G2 O(T(MB$ND@4$%Q2](https://user-images.githubusercontent.com/79007796/176810116-6d6204bc-e8c6-4826-8e31-55f7ba040976.png)
然后，正方提出访问 github 要挂*的问题。然而，我这里可以正常访问 github，倒是开往无法访问。
![2](https://user-images.githubusercontent.com/79007796/176810296-392dcc7b-dd57-4a7f-aeb4-a8b66608b6d0.PNG)
所以，这个属于各地运营商的问题，需要自行解决。

**Reply by imjason** [reply]
各地运营商的问题，所以使用新平台可以更方便使用

**Comment by Jan-York** [comment]
我来说说我的看法吧。
开往是一个公益项目，为加入开往的网站以接力的形式带来私域流量。
当然，主要都是博客网站，handsome作者在更新主题版本时，曾发过一个公告：不要忘记博客的初衷是记录和分享。
在科技飞速发展的今天，或许网络发展太快，博客的圈子一下子就大了起来，很繁华，也很美好，能在一个个个人网站中，看到大千世界。
我很喜欢十年的虫洞和开往这样的程序，他能在你无聊时随机穿越到一个陌生而又繁华的世界(博客)，在逛别人博客的同时，我也会有感触，也会学习到一些东西。
但是，确实，如果要我在十年和开往中选择，我会选择十年，他们有严格的申请模式，有官方论坛，官方网站，甚至说有自己的周边(我觉得这很好，只是不好忘记曾经有一个繁华，叫做十年之约)。他们后台我体验过，快速的审核机制，驳回和通过都可以发送邮件提醒，一键监视博客状态，为已经死亡的博客，新加入的博客，都建立了一个公示，这也很不错。论坛发帖也会有机器人在Q群中通知。这种感觉就很好，个人非常喜欢这样的生态，而开往想必到了这个GitHub地址也都感觉到差距了。

没错，开往确实和十年的初衷不一样，所以我想看看你们的建议。

github说实话，我现在进来都是开着梯子的，很多地方很不方便。

我的想法是为开往构建一个生态，数据不变，审核制度不变，开往的页面也不变。
我们想打造一个开往官方网站，方便审核，订阅，监控判断死亡，不能访问的博客。
想尝试构建交流的圈子，发布公告与官方的推文，创建官方的群聊，也可以试试用uniapp打造一下官方圈子(论坛)App，显得更方便。听起来或许很难，但我还是比较相信这个计划的。
也曾想，互联网某个地方，某个网站，叫做开往，里面存在活跃的博客交流用户，这不重要，重要的是多年以后，还是这些老朋友，人脉的作用不言而喻。
也曾想，发布福利与免费的周边产品，也曾想将更新的文章订阅推送到官方站。
让他不依托Github，而也能显得繁华灿烂。

我想看看大家的意见，开往的用户们。(至于与时同行，我觉得可以改名为开往)。

想一辆火车，总要走走未来的方向。

**Comment by JanYork** [comment]
抱歉，上面那篇用的不是我的github，我换个号发一次..这样比较好。

我来说说我的看法吧。
开往是一个公益项目，为加入开往的网站以接力的形式带来私域流量。
当然，主要都是博客网站，handsome作者在更新主题版本时，曾发过一个公告：不要忘记博客的初衷是记录和分享。
在科技飞速发展的今天，或许网络发展太快，博客的圈子一下子就大了起来，很繁华，也很美好，能在一个个个人网站中，看到大千世界。
我很喜欢十年的虫洞和开往这样的程序，他能在你无聊时随机穿越到一个陌生而又繁华的世界(博客)，在逛别人博客的同时，我也会有感触，也会学习到一些东西。
但是，确实，如果要我在十年和开往中选择，我会选择十年，他们有严格的申请模式，有官方论坛，官方网站，甚至说有自己的周边(我觉得这很好，只是不好忘记曾经有一个繁华，叫做十年之约)。他们后台我体验过，快速的审核机制，驳回和通过都可以发送邮件提醒，一键监视博客状态，为已经死亡的博客，新加入的博客，都建立了一个公示，这也很不错。论坛发帖也会有机器人在Q群中通知。这种感觉就很好，个人非常喜欢这样的生态，而开往想必到了这个GitHub地址也都感觉到差距了。

没错，开往确实和十年的初衷不一样，所以我想看看你们的建议。

github说实话，我现在进来都是开着梯子的，很多地方很不方便。

我的想法是为开往构建一个生态，数据不变，审核制度不变，开往的页面也不变。
我们想打造一个开往官方网站，方便审核，订阅，监控判断死亡，不能访问的博客。
想尝试构建交流的圈子，发布公告与官方的推文，创建官方的群聊，也可以试试用uniapp打造一下官方圈子(论坛)App，显得更方便。听起来或许很难，但我还是比较相信这个计划的。
也曾想，互联网某个地方，某个网站，叫做开往，里面存在活跃的博客交流用户，这不重要，重要的是多年以后，还是这些老朋友，人脉的作用不言而喻。
也曾想，发布福利与免费的周边产品，也曾想将更新的文章订阅推送到官方站。
让他不依托Github，而也能显得繁华灿烂。

我想看看大家的意见，开往的用户们。(至于与时同行，我觉得可以改名为开往)。

想一辆火车，总要走走未来的方向。

**Reply by linlinzzo** [reply]
我理解的意思是您看开往生态不好，看十年之约眼红了。但开往依然在维护互联网的开放，依然在发挥它的作用。我个人也不反对开往与其他项目的合作，只是如果开往与十年之约谈谈会不会给我们节省更多的精力呢？

**Reply by Jan-York** [reply]
如果你要这样理解为眼红，那我并不觉得是这样，还是看看用户的投票在说吧，至于你们如果是与十年合作，那是你们的事情，此次合作也并非我提出的。你也无需再争论了，既然决定投票，那就静待结果

**Comment by linlinzzo** [comment]
![3](https://user-images.githubusercontent.com/79007796/176854652-a9f3d1d7-d08d-4842-a980-9054c35e10c4.PNG)
据图，开往因为**自认为**没有能力与十年之约合作，所以就不和十年之约谈合作。咳咳咳

你们明明知道【与时同行】还不成熟，论坛还在写。那为什么不和已经建立论坛且流量还不错的十年之约谈谈？还要另建一个新论坛（_跟十年之约竞争？_）

**Reply by Jan-York** [reply]
你也可以试试与十年合作，不过貌似他们已经存在虫洞了，如果你们是要换一个强大的合作，那完全可以直接提出来，我们可以放弃争论，因为人家十年生态挺好的，看你们怎么想。如果是想看看投票结果来决定，我也愿意等待

**Reply by linlinzzo** [reply]
> 你也可以试试与十年合作，不过貌似他们已经存在虫洞了，如果你们是要换一个强大的合作，那完全可以直接提出来，我们可以放弃争论，因为人家十年生态挺好的，看你们怎么想。如果是想看看投票结果来决定，我也愿意等待

开往和虫洞不一样吧。

**Reply by Jan-York** [reply]
那你觉得这个接力和虫洞是可以并存的吗，我感觉十年可能只需要虫洞就可以。你可以试试，如果你们和十年合作那肯定比我们现在的情况要好。

**Reply by zmoyi** [reply]
我来说一下吧
第一，不是与时同行这边提出来的合作，并不是与时同行硬性要和开往进行合作，又或者是去蹭开往的流量
第二，与时同行与十年的定位不同，十年侧重点主要在于博客的友联跳转接力，而与时同行侧重点是全类型网站友联的周边接力，又或者说是都是为了公益，为友联接力而做的一些东西，至于一些人说与时同行照抄十年的话，我也表示无奈，因为现阶段新版本还在重构中，也许你们对现版本的与时同行有一些误会
第三，我们初心不变，就是想搞一个好点的生态，如果触碰了一些人的蛋糕，那我们可以选择不合作

**Comment by BH4HPA** [comment]
抱歉，没有很明白 ⌈与时同行⌋ 与 ⌈十年之约⌋ 这俩各是什么项目，看到这个投票的时候一头雾水，可以稍微做下介绍吗？

**Reply by linlinzzo** [reply]
十年之约是一个博主联盟，地址是 https://foreverblog.cn/ 。
与时同行类似于十年之约，地址我这没有，听说在重构。

**Reply by imjason** [reply]
times.sady0.com
